File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        s_count = [0]*26
        for i in range(len(s)):
            s_count[ord(s[i])-ord('a')] += 1
        for i in range(len(t)):
            s_count[ord(t[i])-ord('a')] -= 1
        return [0]*26 == s_count

File: 0242-valid-anagram/0242-valid-anagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function(s, t) {
    if (s.length !== t.length) return false
    
    let hmap = {}
    for (let i = 0; i < s.length; i++) {
        hmap[s[i]] = (hmap[s[i]] || 0) + 1
    }

    for (let i = 0; i < t.length; i++) {
        if (!hmap[t[i]]) return false
        hmap[t[i]] -= 1
    }

    return true
};